src/app/prices/prices.component.ts

```diff
@@ -4,13 +4,59 @@ import { Router } from '@angular/router';
 
 import { FeedbackComponent } from '@ui/feedback';
 
+import { Icontribution } from './interfaces/contribution.prices.interface';
+import { Iprice } from './interfaces/prices.interface';
+
 @Component({
   selector: 'app-prices',
   templateUrl: './prices.component.html',
   styleUrls: ['./prices.component.scss'],
 })
 export class PricesComponent {
   constructor(private router: Router, private readonly _dialog: MatDialog) {}
+  public price: Iprice[] = [
+    {
+      title: 'Полный день в саду',
+      age: 'от 1,5 до 5,5 лет',
+      includedPrice: [
+        '5-ти разовое питание ',
+        'Игровая деятельность',
+        'Прогулки',
+        'Дневной сон ',
+        'Групповые игры на английском языке',
+        'Групповые логопедические игры',
+      ],
+      time: 'C 8-00 до 19-00',
```

Comment:
@MaxOvs19 we need to separate time field. I propose to use time: { from: '8', to: '16:30' } for example